src/css/components/_on-screen-presets.scss

```diff
@@ -0,0 +1,82 @@
+.on-screen-presets { 
+    height: 100%;
+    @include display(flex);
+    @include flex-direction(column);
+}
+
+.on-screen-presets .album-art {
+    width: 35%;
+    height: 100% !important;
+    max-height: 100%;
+    display: flex;
+    justify-content: center;
+    transform: translateY(0%);
+    padding: 10px;
+    align-items: center;
+    margin: auto;
+
+    img {
+        width: 100%;
+    }
+}
```

Comment:
I think this could be generalized to both templates, just edit the following in `_album-art.scss`:
```
@mixin album-art-size() {
    $calc-width: $master-width / 2.954;
    $calc-height: ($master-height - 75px)  / 1.8461;
    @if $calc-width <= $calc-height {
        $size: $calc-width + ($calc-width / 6.5);
        @include size($size);
    } @else {
        $size: $calc-height + ($calc-height / 6.5);
        @include size($size);
    }
}

.album-art {
    @include display(flex);
    @include align-items(center);
    margin: auto 0px;
    @include album-art-size();
    height: 100% !important
    border-radius: 4px;

    img {
        max-width: 100%;
        max-height: 100%;
    }
}

.album-art-background{
    height: 100%;
    width: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
}
```
This makes it so the image stays in the bounds of the container properly in both templates, with no overlap with the progress bar (you could also add padding in if you think it looks better)